templates/orders/details.php

```diff
@@ -238,6 +238,7 @@
                                 <span><?php esc_html_e( 'Customer IP:', 'dokan-lite' ); ?></span>
                                 <?php echo esc_html( get_post_meta( dokan_get_prop( $order, 'id' ), '_customer_ip_address', true ) ); ?>
                             </li>
+                            <?php apply_filters( 'dokan_manage_order_details_after', $order, $order_id, $order_items);?>
```

Comment:
Please change this filter name to `dokan_order_details_after_customer_info` instead of `dokan_manage_order_details_after`

Reply:
resolved